nautobot/core/settings.py

```diff
@@ -825,3 +825,6 @@ def UI_RACK_VIEW_TRUNCATE_FUNCTION(device_display_name):
     "TimeZoneSerializerField": {"type": "string"},
     "UUIDField": {"type": "string", "format": "uuid"},
 }
+
+
+FEEDBACK_BUTTON_ENABLED = True
```

Comment:
Also need to add documentation of this setting in `core/templates/nautobot_config.py.j2`